tutorials/creating_sequences.ipynb

```diff
@@ -4,543 +4,308 @@
     "\n",
-    "qubits = {f\"q{i}\": point for (i, point) in enumerate(square)}\n",
-    "reg = pulser.Register(qubits)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "The `Register` class provides some useful features, like the ability to visualise the array and to make a rotated copy."
+    "This tutorial illustrates the recipe to program using Pulser provided in the [programming page](docs/programming.md). For more information regarding the steps followed and the mathematical objects at stake, please refer to this page."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(\"The original array:\")\n",
-    "reg.draw()\n",
-    "reg1 = reg.rotated(45)  # Rotate by 45 degrees\n",
-    "print(\"The rotated array:\")\n",
-    "reg1.draw()"
+    "import numpy as np\n",
+    "import pulser"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If one doesn't particularly care about the name given to the qubits, one can also create a `Register` just from a list of coordinates (using the `Register.from_coordinates` class method). In this case, the qubit ID's are just numbered, starting from 0, in the order they are provided in, with the option of adding a common prefix before each number. Also, it automatically centers the entire array around the origin, an option that can be disabled if desired."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "reg2 = pulser.Register.from_coordinates(\n",
-    "    square, prefix=\"q\"\n",
-    ")  # All qubit IDs will start with 'q'\n",
-    "reg2.draw()"
+    "## Programming a constant Hamiltonian without interaction: Exciting one atom in the Rydberg state"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Furthermore, there are also built-in class methods from creation of common array patterns, namely:\n",
-    "- Square lattices in rectangular or square shapes\n",
-    "- Triangular lattices\n",
+    "As presented in [\"Programming a neutral-atom QPU\"](docs/programming.md), Pulser enables you to program [an Hamiltonian $H$](docs/programming.md#2-hamiltonian-evolves-the-state) composed of an interaction Hamiltonian and a drive Hamiltonian.\n",
     "\n",
-    "We could, thus, create the same square array as before by doing:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "reg3 = pulser.Register.square(\n",
-    "    4, spacing=5, prefix=\"q\"\n",
-    ")  # 4x4 array with atoms 5 um apart\n",
-    "reg3.draw()"
+    "Let's program this Hamiltonian such that an atom initially in the ground state $\\left|g\\right>$ is measured in the rydberg state $\\left|r\\right>$ after a time $\\Delta t$. "
```

Comment:
I think it would make sense to write down the drive Hamiltonian we will program and why the interaction term is zero in this case before moving to the recipe steps. 
In general, it sounds most logical me to first define what we want to program and then execute